themes/jb/layouts/partials/episode/schedule.html

```diff
@@ -0,0 +1,22 @@
+<div class="card card-episode">
+
+  <div class="card-image">
+    <a href="{{ .Permalink }}">
+      <img src="{{.Params.header_image }}" width="432" height="242" alt="{{.Title}}">
+    </a>
+  </div>
+  <div class="card-content">
+     <div class="content">
+      <a href="{{ .Permalink }}"><h3>{{.Title}}</h3></a>
+      <p class="schedule"
+          data-live-schedule-type="{{ index .Params.live.type }}"
+          data-live-schedule-day="{{ index .Params.live.day }}"
+          data-live-schedule-hour="{{ index .Params.live.hour }}"
+          {{ if (isset .Params.live "minute") }}
+            data-live-schedule-minute="{{ index .Params.live.minute }}"
+          {{ end }}
+      ></p>
+      </div>
+  </div>
+
+</div>
```

Comment:
Instead of multiple `a` tags for the image and the title, wrap the entire card in it so the entire card is a link to the show. This will match the functionality of other cards on the site

```suggestion
<a href="{{ .Permalink }}">
  <div class="card card-episode">
    <div class="card-image">
      <img src="{{.Params.header_image }}" width="432" height="242" alt="{{.Title}}">
    </div>
    <div class="card-content">
      <div class="content">
        <h3>{{.Title}}</h3>
        <p class="schedule"
           data-live-schedule-type="{{ index .Params.live.type }}"
           data-live-schedule-day="{{ index .Params.live.day }}"
           data-live-schedule-hour="{{ index .Params.live.hour }}"
           {{ if (isset .Params.live "minute") }}
             data-live-schedule-minute="{{ index .Params.live.minute }}"
          {{ end }}></p>
      </div>
    </div>
  </div>
</a>
```